components/style/stylesheets.rs

```diff
@@ -76,21 +70,24 @@ impl From<SingleRuleParseError> for RulesMutateError {
     }
 }
 
+fn only_ns_or_import(rules: &[CssRule]) -> bool {
```

Comment:
Now that CssRules is a plain vec, you can make this a `&self` method  on `CSSRules` itself. It wasn't before because it wouldn't be possible to handle the locking in the method.